modules/meeting/app/components/meetings/update_flash_component.sass

```diff
@@ -0,0 +1,4 @@
+@import 'helpers'
+
+.op-meeting-update-flash
```

Comment:
Hi @HDinger, could I get your opinion on this? I want to override the default width to prevent a new line in the flash message. Is this an okay way to do that?

Reply:
Hi @mrmir 

I see two potential issues with that:

1. I don't know the exact use case but when doing such an optimization to avoid line breaks, there is usally the question of other languages. Most likely there will be at least one language that is longer and thus the line will break nevertheless. Is that accepted? 
2. This is a very specific rule. However, I can imagine that there are other places that want a wider flash message. So maybe a generic `-wide` class would be better.

Reply:
While thinking about it, I am wondering why there is the need to avoid line breaks overall. We have that already in other flashes, so why is there the need to avoid that here?

Reply:
Hmm, I was just trying to get it to look like the specification, but the point about translations messing it up anyway is something I completely overlooked. I will leave it as default, thanks!